Configuration/Skimming/python/autoSkim.py

```diff
@@ -41,6 +41,10 @@
  'SingleMuon': 'LogError+LogErrorMonitor',
 }
 
+# For 2023 PbPb skims
+for i_split in range(32):
+    autoSkim[f'HIPhysics{i_split}'] = 'PbPbEMu+PbPbZEE+PbPbZMM+LogError+LogErrorMonitor'
```

Comment:
The dataset name should be `HIPhysicsRawPrime`

Reply:
Thanks for the reminder. Let's wait for the final decision from TSG.

Reply:
Sigh.
@cms-sw/hlt-l2 please confirm that we are running raw prime

Reply:
Is this still a matter of discussion? I have missed the relevant thread / pointer @youyingli

Reply:
> Sigh. https://github.com/orgs/cms-sw/teams/hlt-l2 please confirm that we are running raw prime

I'm not sure what this comment is about. `HIPhysicsRawPrime*` was suggested by TSG and never questioned (if it was for the menus provided by HIon HLT contacts, this would still be called "HITestRawPrime*".. sigh).

Reply:
This thread just wants to make sure if the PD name for 2023 PbPb PromptReco will be ```HIPhysicsRawPrime``` with 32 splittings (i.e. HIPhysicsRawPrime0, HIPhysicsRawPrime1, ..., HIPhysicsRawPrime31) containing HLT ```HLT_HIEle*Gsf_v*", "HLT_HIL3SingleMu*_Open_v*```. @missirol , could you confirm that? This is relevant to what PDs should have PbPb skim format.

Reply:
> > Sigh. https://github.com/orgs/cms-sw/teams/hlt-l2 please confirm that we are running raw prime
> 
> I'm not sure what this comment is about. `HIPhysicsRawPrime*` was suggested by TSG and never questioned (if it was for the menus provided by HIon HLT contacts, this would still be called `HITestRawPrime*`.. sigh).
> 
> Ref: [CMSHLT-2882](https://its.cern.ch/jira/browse/CMSHLT-2882)

Well, we told Youling that the correct name is HIPhysicsRawPrime*, but apparently Youling would like to hear it from TSG.

Reply:
See this comment:  https://github.com/cms-sw/cmssw/pull/42754#discussion_r1322388598